user/xournalpp/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "xournalpp"
+pkgver = "1.2.3"
+pkgrel = 0
+archs = ["aarch64", "x86_64"]
+build_style = "cmake"
+configure_args = [
+    "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+]
+hostmakedepends = ["cmake", "pkgconf", "gettext", "ninja"]
+makedepends = [
+    "glib-devel",
+    "gtk+3-devel",
+    "poppler-devel",
+    "libxml2-devel",
+    "libzip-devel",
+    "portaudio-devel",
+    "libsndfile-devel",
+    "librsvg-devel",
+    "libzip-devel",
+    "alsa-lib-devel",
```

Comment:
```suggestion
```

not used

Reply:
(it fails to link if you remove it but that's a bug in portaudio, i'll apply that change for you so after rebase it will work)

Reply:
there, added in 03dc80d0641062ebade918b80c67da6235873dc6 so you can drop that commit and rebase